web/views.py

```
@@ -2189,13 +2190,21 @@ def student_dashboard(request):
     # Query achievements for the user.
     achievements = Achievement.objects.filter(student=request.user).order_by("-awarded_at")
 
+    # Get user's membership subscription
+    subscription = None
+    try:
+        subscription = UserMembership.objects.get(user=request.user)
+    except UserMembership.DoesNotExist:
+        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(UserMembership.DoesNotExist)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(UserMembership.DoesNotExist)`

(SIM105)